recipes/speex/meta.yaml

```diff
@@ -0,0 +1,39 @@
+{% set version = "1.2rc2" %}
```

Comment:
@jakirkham, @183amir is right with the speex version. However, I can test also the 1.2.0 release, and if it works with Kaldi, I can update the recipe.